src/sentry/static/sentry/app/views/projectKeyDetails.jsx

```diff
@@ -159,6 +311,73 @@ const KeySettings = React.createClass({
           </div>
         </div>
 
+        {!rateLimitsEnabled
+          ? this.state.hooksDisabled
+              .map(hook => {
+                return hook(organization, project, data);
+              })
+              .find(() => true)
```

Comment:
```javascript
.find(() => true)
```

Won't this just always return the first one? Is this intended to be a "safe" way of accessing the first index, if it exists (nothing otherwise)? Could you not use `Array.prototype.shift` instead (seems clearer and equally safe - returns `undefind` if the array is empty)?